### PR TITLE
TDR-3474 - Backfill new columns of the Consignment table

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

**Review by TomJKing — COMMENTED** [review]
Looks good. One comment. Also assume both columns are now being populated correctly via the api?

File: lambda/src/main/resources/db/migration/V135__backfill_new_columns_of_consignment.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Backfill new columns of the Consignment table ("SeriesName" and "TransferringBodyName")
```
[the remark this answers is not in the document]
Comment (vimleshtna) [reply lambda/src/main/resources/db/migration/V135__backfill_new_columns_of_consignment.sql:1]: Yeah, good point. Update it. Thanks

**Comment by vimleshtna** [comment]
> Looks good. One comment. Also assume both columns are now being populated correctly via the api?

Yes, both columns are being populated when we start a new transfer and select a series.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks. Once this has been run on intg do a manual check that its populated the columns correctly.

**Pull request merged**